src/sandbox/sandbox_linux.go

```diff
@@ -67,6 +69,29 @@ func Sandbox(args []string) error {
 			return fmt.Errorf("Failed to bring loopback interface up: %s", err)
 		}
 	}
+
+	if user != "" {
+		userID, err := strconv.Atoi(user)
+		if err != nil {
+			return fmt.Errorf("invalid SANDBOX_UID: %v", user)
+		}
+		execCmd := exec.Command(cmd, args[1:]...)
+		execCmd.Env = env
+		execCmd.Stdout = os.Stdout
+		execCmd.Stdin = os.Stdin
+		execCmd.Stderr = os.Stderr
+		execCmd.SysProcAttr = &syscall.SysProcAttr{
+			Pdeathsig:  syscall.SIGHUP,
```

Comment:
Maybe SIGTERM or SIGINT instead?

Reply:
I _think_ this is the signal Please sends to its sub-processes when the process is cleaned up by it's process manager. It does seem like the process group is cleaned up correctly.

Reply:
Right, I misunderstood the `Pdeathsig` documentation. Thanks for clarifying!